msgraph/users.go

```diff
@@ -437,3 +437,23 @@ func (c *UsersClient) DeleteManager(ctx context.Context, id string) (int, error)
 
 	return status, nil
 }
+
+// UploadThumbnailPhoto uploads a thumbnail photo for the specified user which should be a gif, jpeg or png image.
+func (c *UsersClient) UploadThumbnailPhoto(ctx context.Context, userId, contentType string, thumbnailData []byte) (int, error) {
+	var status int
```

Comment:
I think this line can be removed. as `status` is also defined bellow.

Reply:
The syntax and format was largely copied/rewritten from: 

https://github.com/manicminer/hamilton/blob/c99311563ba9c4bf460ff28609eb303769d20d00/msgraph/applications.go#L608

If that has the status initialized before that function then my guess is the HTTP call can sometimes return no error but a function. Am I wrong in how that would work?

Reply:
The `:=` operator creates the required variables.

That `var status int` declaration is not required.

If you look [at the next function, at the file you've copied it from](https://github.com/manicminer/hamilton/blob/c99311563ba9c4bf460ff28609eb303769d20d00/msgraph/applications.go#L629), you'll see it does not uses that declaration.

Reply:
I figured that `var status int` was something I didn't understand about the HTTP returns etc. Must be legacy code.

I will remove for this function.